pkg/apis/stork/v1alpha1/applicationbackup.go

```diff
@@ -110,6 +110,8 @@ type ApplicationBackupVolumeInfo struct {
 	StorageClass             string                      `json:"storageClass"`
 	Provisioner              string                      `json:"provisioner"`
 	VolumeSnapshot           string                      `json:"volumeSnapshot"`
+	Uid                      int64                       `json:"uid"`
```

Comment:
Could you use a new Structure object to store security Context details. Today its just UID and GID but tomorrow we may end up storing more information like labels.
And since these fields are used only for PSA enabled clusters, for non PSA these are unused, it makes more sense to capsulate the fields into relevant structure.

Reply:
done